lib/src/controls/inputs/dropdown_button.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:fluent_ui/fluent_ui.dart';
+
+const double _kDefaultPadding = 5.0;
+const double _kVerticalOffset = 20.0;
+const double _kContentWidth = 100.0;
+
+class DropDownButton extends StatelessWidget {
+  const DropDownButton({
+    Key? key,
+    required this.items,
+    required this.controller,
+    this.leading,
+    this.title,
+    this.trailing,
+    this.padding,
+    this.verticalOffset,
+    this.contentWidth,
+    this.closeAfterClick = true,
+    this.disabled = false,
+    this.focusNode,
+    this.autofocus = false,
+  }) : super(key: key);
+
+  final Widget? leading;
+  final Widget? title;
+  final Widget? trailing;
+  final EdgeInsets? padding;
+  final double? verticalOffset;
+  final double? contentWidth;
+  final List<DropDownButtonItem> items;
+  final FlyoutController controller;
+  final bool closeAfterClick;
+  final bool disabled;
+
+  /// {@macro flutter.widgets.Focus.focusNode}
+  final FocusNode? focusNode;
+
+  /// {@macro flutter.widgets.Focus.autofocus}
+  final bool autofocus;
+
+  @override
+  Widget build(BuildContext context) {
+    assert(debugCheckHasFluentTheme(context));
+
+    final buttonChildren = <Widget>[];
+    if (leading != null) {
+      buttonChildren.add(Padding(
+        padding: EdgeInsets.only(right: padding?.right ?? _kDefaultPadding),
+        child: leading,
+      ));
+    }
+    if (title != null) {
+      buttonChildren.add(title!);
+    }
+    buttonChildren.add(Padding(
+      padding: EdgeInsets.only(left: padding?.left ?? _kDefaultPadding),
+      child: trailing ?? const Icon(FluentIcons.chevron_down, size: 12),
+    ));
```

Comment:
For better readability:
```suggestion
    final buttonChildren = <Widget>[
      if (leading != null)
        Padding(
          padding: EdgeInsets.only(right: padding?.right ?? _kDefaultPadding),
          child: leading,
        ),
      if (title != null) title!,
      Padding(
        padding: EdgeInsets.only(left: padding?.left ?? _kDefaultPadding),
        child: trailing ?? const Icon(FluentIcons.chevron_down, size: 12),
      ),
    ];
```